src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Color {
+    Blue,
+    Red,
+    Yellow,
+    Green,
+    Black

```

Comment:
Enum constants should be written in uppercase letters. For example, 'BLUE' instead of 'Blue'. This follows the Java naming conventions for enum values.